LayoutTests/imported/w3c/web-platform-tests/cookie-store/cookieStore_third_party_iframe_change_listener.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<meta charset='utf-8'>
+<script src='/resources/testharness.js'></script>
+<script src='/resources/testharnessreport.js'></script>
+<iframe id='iframe' src='http://127.0.0.1:8800/cookie-store/resources/third_party_iframe_change_listener.html'></iframe>
```

Comment:
Since his is a WPT test, you shouldn't be hard-coding host names / ports, that's not OK outside WebKit-only tests.

You'll have to use get-host-info.sub.js and `get_host_info()`. You can see examples in other WPT tests.

Reply:
But also, cookie partitioning (or blocking) in third-party iframes is not consistent across browsers so it may just be safer to move this test to http/tests/wpt.

Reply:
Could be worth making a .tentative.html test and exporting to WPT